release-announcements/20210701-yt-4.0.0.txt

```diff
@@ -0,0 +1,82 @@
+The yt community is proud to announce the release of yt-4.0!
+
+yt (http://yt-project.org) is an open source, community-developed toolkit for analysis and visualization of volumetric data.
+
+If you are interested in contributing to yt, please see our repository on GitHub at http://github.com/yt-project/yt
+
+This release features a complete reworking of a number of internal components,
+most importantly those related to particles and discrete points.
+
+Information about "what's new" can be found at: https://yt-project.org/docs/4.0.0/yt4differences.html .
+
+A set of highlights of the pull requests that went into this release can be found here:
+
+https://gist.github.com/matthewturk/7a1f21d98aa5188de7645eda082ce4e6#file-highlights-md
+
+A full listing of the pull requests that went into this release can be found here:
+
+https://gist.github.com/matthewturk/7a1f21d98aa5188de7645eda082ce4e6#file-yt4-changelog-md
+
+This release has been years in the making, and features contributions from dozens of individuals, many of whom were new to the project.  It marks the culmination of a set of wide-reaching infrastructure changes that result in much higher fidelity to the underlying data for particle datasets, and we anticipate that improvements to the overall project and project infruastructure will lead directly to a much more rapid release cycle of enhancements and bug fixes in the future.
```

Comment:
Is the last part of this segment important to put in here? Maybe we shouldn't promise it.

Reply:
Well, I think our release cycle has been really slow, in part because of the length of this release.  I thought it would be worthwhile to mention that we do not expect bug fixes to take a long time to roll out in the future.  I can remove it if you like.